src/lib/Layout/PanelSideBarLayout/PanelSideBar/Definitions/PanelItem.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/named
```

Comment:
why is this eslint needed? Something seems off here.

Reply:
i dont remember how and why, but i was able to solve it 😄